board/safety/safety_toyota.h

```diff
@@ -12,7 +12,9 @@
 
 #define TOYOTA_COMMON_SECOC_TX_MSGS \
   TOYOTA_BASE_TX_MSGS \
-  {0x2E4, 0, 8}, {0x131, 0, 8}, \
+  {0x2E4, 0, 8}, {0x131, 0, 8},  /* STEERING_LKA (longer message for SecOC), STEERING_LTA_2 */  \
+  {0x183, 0, 8}, {0x411, 0, 8},  /* ACC_CONTROL_2, PCS_HUD */  \
+  {0x750, 0, 8},  /* radar diagnostic address */  \
```

Comment:
Remove `0x411` and `0x750`, the code from `opendbc` is currently not sending these 2 messages. We don't disable radar to enable openpilot longitudinal on this car.
```suggestion
  {0x2E4, 0, 8}, {0x131, 0, 8},  /* STEERING_LKA (longer message for SecOC), STEERING_LTA_2 */  \
  {0x183, 0, 8},  /* ACC_CONTROL_2 */  \
```